helm_deploy/values-dev.yaml

```diff
@@ -4,16 +4,16 @@
 replicaCount: 2
 
 image:
-  repository: quay.io/hmpps/calculate-people-move-prices
+  repository: quay.io/hmpps/calculate-journey-variable-payments
   tag: latest
   port: 8080
 
 ingress:
   enabled: true
   enable_whitelist: true
   hosts:
-    - host: calculate-people-move-prices-dev.hmpps.service.justice.gov.uk
-      cert_secret: calculate-people-move-prices-cert
+    - host: calculate-journey-variable-payments-staging.hmpps.service.justice.gov.uk
```

Comment:
I understand the need for this but will probably cause confusion later on down the line...not a lot we do about it now.